rr_iarrc/launch/test_segnet_and_planner.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+    <include file="$(dirname)/line_detector_segnet.launch"/>
```

Comment:
why $(dirname) rather than package relative?

Reply:
Purely because it's shorter. Is it best practice to use package-relative even when the included file is in the same folder?